tools/testsys/src/aws_resources.rs

```diff
@@ -10,24 +10,24 @@ use aws_sdk_ec2::Region;
 use bottlerocket_variant::Variant;
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::ObjectMeta;
 use k8s_openapi::serde_json::Value;
+use log::debug;
 use maplit::btreemap;
 use model::constants::NAMESPACE;
 use model::{
     Agent, Configuration, Crd, DestructionPolicy, Resource, ResourceSpec, SecretName, Test,
     TestSpec,
 };
 use std::collections::BTreeMap;
+use testsys_config::{
+    rendered_cluster_name, AwsEcsVariantConfig, AwsK8sVariantConfig, TestsysImages,
+};
 
 pub(crate) struct AwsK8s {
```

Comment:
I'll leave this file's large diff alone, but I'm flagging it as another place that we will have to think about extensibility for out-of-tree variants.